Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -28,10 +29,50 @@
 
 namespace ROS2
 {
+    bool CollidersMaker::GuessIsWheel(const AZStd::string& entityName)
+    {
+        AZStd::regex wheel_regex("(?i)wheel|(?i)wh_|(?i)(_wh)");
+        AZStd::smatch match;
+        if (AZStd::regex_search(entityName, match, wheel_regex))
+        {
+            AZ_Printf("WheelMaterial", "%s is a wheel\n", entityName.c_str());
+            return true;
+        }
+        AZ_Printf("WheelMaterial", "%s is NOT a wheel\n", entityName.c_str());
+        return false;
+    }
+
     CollidersMaker::CollidersMaker(AZStd::string modelPath)
         : m_modelPath(AZStd::move(modelPath))
         , m_stopBuildFlag(false)
     {
+        // get Wheel material from assets
+        bool assetFound = false;
+        AZ::Data::AssetInfo assetInfo;
+        AZStd::string watchDir;
```

Comment:
unnecessary?

Reply:
Unfortunately it is needed to provide reference to string here :(